bls/src/main/java/tech/pegasys/teku/bls/impl/SecretKeyInterface.java

```diff
@@ -17,7 +17,7 @@
 import org.apache.tuweni.bytes.Bytes32;
 
 /** This class represents a BLS12-381 private key. */
-public interface SecretKey {
+public interface SecretKeyInterface {
```

Comment:
Do we really need this refactor?

Reply:
The supranational blst now has a class called SecretKey, so I changed the name of our interface to not interfere with that one.

Reply:
May be just refer to BLST `SecretKey` via full name when needed?

Reply:
`SecretKeyInterface` looks a bit weird among `PublicKey`, `Signature`, `KeyPair`

Reply:
Like here https://github.com/ConsenSys/teku/blob/ae994ca4538eba3d7063fb4990277385f75b26c2/data/serializer/src/main/java/tech/pegasys/teku/api/schema/BeaconBlock.java#L62-L65

I don't think this is a good style having same class names across different packages, but if a class name in another lib has accidentally the same name as ours there is nothing bad just to refer it with the full qualified name

Reply:
Makes sense. Done.